classes/local/store/object_file_system.php

```diff
@@ -365,29 +371,96 @@ public function delete_empty_folders($rootpath = null) {
             $emsg = $e->getMessage();
             $path = substr($emsg, strrpos($emsg, '(') + 1, strrpos($emsg, ')') - 1);
             $msg = substr($emsg, strrpos($emsg, ':') + 2);
-            $message = "Delete empty dir failed: $path $msg {$this->get_path_owner_info($path)}";
+            $message = "Delete empty dir failed: $path $msg";
```

Comment:
Did this leak from the other patch?

Reply:
yes, but I'll get removed if we merged https://github.com/catalyst/moodle-tool_objectfs/pull/220